examples/internet-gateway/main.tf

```diff
@@ -20,7 +20,6 @@ module "gateway" {
 
   # Attach the Gateways to your VPC and subnets.
   vpc            = aws_vpc.main.id
-  public_subnet  = aws_subnet.public.id
```

Comment:
If this is now using an Internet Gateway, this can only be put in a public subnet right?

Reply:
The example is, yes. That's why our AWS guide walks the user through.

The top-level module doesn't require a public subnet though.

Reply:
Wait, so you're saying the next line below (`aws_subnet.private.id`) is actually a public subnet?

Reply:
The private subnet is used to connect to the instance via EC2 instance connect, further below.

The public subnet is not associated in this example module, but it is in the top-level module when associated the elastic IP addresses.

I.e. the `public_subnet` variable was updated to `attach_public_subnet` bool.